common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -166,6 +167,10 @@ def get_static_transcript(self, request, transcripts):
         if not self.transcript_language == 'en':
             return response
 
+        # If this video lives in library, the code below is not relevant and will error.
```

Comment:
Right now there is no support for static assets in libraries. But I think that in the future it would make sense to add static asset support. There are a lot of use cases it would enable, including this one. Unfortunately it's a non-trivial change, and would require some fancy routing code when library content requests static assets in a course context.